### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by rjeberhard**

### CODE REVIEW

File: docs-source/content/release-notes.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ draft: false
 
 | Date | Version | Introduces backward incompatibilities? | Change |
 | --- | --- | --- | --- |
-| March ??, 2021 | v3.2.0 | no | Placeholder |
+| March 26, 2021 | v3.2.0 | no | Online updates for dynamic changes for Model in Image, injection of the WebLogic Monitoring Exporter, other features, and significant number of additional fixes. |
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:11]: and significant -> and a significant

File: docs-source/content/release-notes.md [hunk 2]
```diff
@@ -39,6 +39,37 @@ draft: false
 
 #### Operator 3.2.0
 
+##### Features
+
+* The operator's container image is based on Oracle Linux 8.
+* WebLogic Server container images based on Oracle Linux 8 are supported.  
+* [Online updates]({{<relref "/userguide/managing-domains/model-in-image/runtime-updates/_index.md#online-updates">}}) of dynamic configuration changes for Model in Image.
+* Automatic injection of the [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter) as a sidecar container.  
+* [Events]({{< relref "/userguide/managing-domains/domain-events.md" >}}) are generated at important moments in the lifecycle of the operator or a domain.
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:48]: lifecycle  -> life cycle (life cycle (n), lifecycle (adj))

File: docs-source/content/release-notes.md [hunk 3]
```diff
@@ -39,6 +39,37 @@ draft: false
...
+* [PodDisruptionBudgets]({{<relref "/userguide/introduction/architecture/_index.md#draining-a-node-and-poddisruptionbudget">}}) are generated for clusters improving the ability to maintain cluster availability during planned node shutdowns and Kubernetes upgrade.
+* Additional scripts to assist with common tasks, such as the scaleCluster.sh script.
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:50]: scaleCluster.sh -> `scaleCluster.sh` (code font)

File: docs-source/content/release-notes.md [hunk 4]
```diff
@@ -39,6 +39,37 @@ draft: false
...
+* Support for TCP and UDP on the same channel when the SIP protocol is used.
+
+##### Fix for Bug or Regression
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:53]: Fix for Bug or Regression -> (We prefer most titles to be plural, therefore, Fixes for Bugs or Regressions. Other suggestion is Fixed Bugs or Regressions.)

File: docs-source/content/release-notes.md [hunk 5]
```diff
@@ -39,6 +39,37 @@ draft: false
...
+
+* All fixes included in 3.1.1 through 3.1.4 are included in 3.2.0.
+* Resolved an issue where clustered managed servers would not start when the administration server was not running ([#2093](https://github.com/oracle/weblogic-kubernetes-operator/pull/2093)).
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:56]: managed servers -> Managed Servers
administration server -> Administration Server

File: docs-source/content/release-notes.md [hunk 6]
```diff
@@ -39,6 +39,37 @@ draft: false
...
+* Model in Image generated domain home file systems that exceed 1 MB are supported ([#2095](https://github.com/oracle/weblogic-kubernetes-operator/pull/2095)).
+* An event and status update are generated when a cluster can't be scaled past the cluster's maximum size ([#2097](https://github.com/oracle/weblogic-kubernetes-operator/pull/2097)).
+* Improve the operator's ability to recover from failures during its initialization ([#2118](https://github.com/oracle/weblogic-kubernetes-operator/pull/2118)).
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:59]: Improve  -> Improved

File: docs-source/content/release-notes.md [hunk 7]
```diff
@@ -39,6 +39,37 @@ draft: false
...
+* Improve the ability for scalingAction.sh to discover the latest API version ([#2130](https://github.com/oracle/weblogic-kubernetes-operator/pull/2130)).
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:60]: Improve -> Improved
scalingAction.sh -> `scalingAction.sh`

File: docs-source/content/release-notes.md [hunk 8]
```diff
@@ -39,6 +39,37 @@ draft: false
...
+* Resolved an issue where the operator's log would show inappropriate warnings related to missing RBAC permissions ([#2138](https://github.com/oracle/weblogic-kubernetes-operator/pull/2138)).
+* Capture WDT errors related to validating the model ([#2140](https://github.com/oracle/weblogic-kubernetes-operator/pull/2140)).
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:62]: Capture  -> Captured

File: docs-source/content/release-notes.md [hunk 9]
```diff
@@ -39,6 +39,37 @@ draft: false
...
+* Resolved an issue where the operator incorrectly missed Secrets or ConfigMaps in namespaces with a large number of either resource ([#2199](https://github.com/oracle/weblogic-kubernetes-operator/pull/2199)).
+* Resolved an issue where the operator could report incorrect information about an introspection job that failed ([#2201](https://github.com/oracle/weblogic-kubernetes-operator/pull/2201)).
+* Resolved an issue where a Service with the older naming pattern from operator 3.0.x could be stranded ([#2208](https://github.com/oracle/weblogic-kubernetes-operator/pull/2208)).
+* Resolved an issue in the domain and cluster start scripts related to overrides at specific managed servers ([#2222](https://github.com/oracle/weblogic-kubernetes-operator/pull/2222)).
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:66]: managed servers -> Managed Servers

**Review by rosemarymarano — COMMENTED** [review]
Few edits

File: docs-source/content/release-notes.md (hunk 8 above)
Comment (jshum2479) [reply docs-source/content/release-notes.md:62]: Is inappropriate the correct word?  or superfluous closer to describe the behavior?

Comment (rjeberhard) [reply docs-source/content/release-notes.md:62]: The issue fixed here is that while the operator was checking if it had appropriate permissions in each namespace, it was checking for permissions in the operator's own namespace that would only be needed in a managed namespace. So, the warnings were incorrect or misleading. I'd be okay with either of those words, but the warnings were truly wrong.

**Review by jshum2479 — APPROVED** [review]
LGTM

**Pull request merged**